owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -46,60 +46,10 @@ class ReleaseNotesViewModel(
     companion object {
         val releaseNotesList = listOf(
             ReleaseNote(
-                title = R.string.release_notes_4_2_0_title_1,
-                subtitle = R.string.release_notes_4_2_0_subtitle_1,
+                title = R.string.release_notes_4_3_0_title_2,
+                subtitle = R.string.release_notes_4_3_0_subtitle_2,
                 type = ReleaseNoteType.CHANGE,
```

Comment:
More than change, I would say it is an enhancement because this is done automatically now

Reply:
Done